lib/libs/preferences.dart

```diff
@@ -240,8 +251,118 @@ class Preferences extends SerializableObject with ChangeNotifier {
+                  child: const Text("Trotzdem fortfahren"),
+                ),
+                TextButton(
+                  onPressed: () {
+                    Navigator.pop(context);
+                    abort = true;
+                  },
+                  child: const Text("Abbrechen"),
+                ),
+              ]
+            ),
+          );
+          if (abort) return "abort";
+        }
+        loadFromJson(importJson["prefs_json"].toString());
+        return "success";
+      } catch (e, s) {
+        logCatch("prefs_import", e, s);
+        return "import_error";
+      }
+    } else {
+      return "abort";
+    }
+  }
+
   Preferences() {
     objectCreators["time_to_next_plan"] = (_) => HMTime(14, 45);
     objectCreators["stuplan_names"] = (_) => <String>[];
   }
 }
+
+Widget Function(BuildContext) sharePreferencesPageBuilder(String? data) =>
+        (context) => SharePreferencesPage(data!);
+class SharePreferencesPage extends StatefulWidget {
+  final String data;
+  const SharePreferencesPage(this.data, {super.key});
+  @override
+  State<SharePreferencesPage> createState() => _SharePreferencesPageState();
+}
+class _SharePreferencesPageState extends State<SharePreferencesPage> {
+  @override
+  Widget build(BuildContext context) {
+    String prefsJson = sharedPreferences.getString(prefsPrefKey) as String;
+    var exportJsonText = {
+      'prefs_version': prefsVersion,
+      'prefs_json': prefsJson,
+    };
+    var exportJson = jsonEncode(exportJsonText);
+    return Scaffold(
+      appBar: AppBar(title: const Text("Gespeicherte Einstellungen")),
+      body: Column(
+        children: [
+          ElevatedButton(
+            onPressed: () {
+              Share.shareXFiles(
+                  [XFile.fromData(utf8.encode(exportJson.toString()), mimeType: 'application/json')], fileNameOverrides: ['Kepler_App_Einstellungen_Export.json'],
+                  sharePositionOrigin: Rect.fromLTWH(
+                      0, 0,
+                      MediaQuery.of(this.context).size.width,
+                      MediaQuery.of(this.context).size.height / 2
+                  )
+              );
+            },
+            child: const Row(
+              mainAxisSize: MainAxisSize.min,
+              children: [
+                Flexible(flex: 0, child: Text("Exportieren")),
+                Flexible(
+                  child: Padding(
+                    padding: EdgeInsets.only(left: 4),
+                    child: Icon(Icons.ios_share, size: 16),
+                  ),
+                ),
+              ],
+            ),
+          ),
+          Expanded(
+            child: SizedBox(
+              width: double.infinity,
+              child: SingleChildScrollView(
+                child: Padding(
+                  padding: const EdgeInsets.all(4),
+                  //child: Text(widget.data + (sharedPreferences.getString(prefsPrefKey) as String)),
+                  child: Text(exportJsonText.toString()),
```

Comment:
Warum willst du den Inhalt gerne anzeigen? Ich verstehe, dass es interessant anzuschauen ist, aber für als Benutzer kannst du damit ja nichts machen bzw. nichts ändern oder so.

Falls du das trotzdem anzeigen willst, bitte als schönes/eingerücktes JSON formatieren und mit einer Monospace-Font anzeigen, z.B.
```txt
{
   "prefs_version": 1,
   ...
}
```

Reply:
War vor allem zum Debuggen gedacht, ich habe mittelfristig vor, zum besseren Debugging in der stabilen Version Entwicklereinstellungen zu implementieren.
In dieser Version macht das zugegebenermaßen wenig Sinn.